fix/fixer_qemu_host_port.go

```diff
@@ -0,0 +1,57 @@
+package fix
+
+import (
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerQEMUHostPort updates ssh_host_port_min and ssh_host_port_max to host_port_min and host_port_max for QEMU builders
+type FixerQEMUHostPort struct{}
+
+func (FixerQEMUHostPort) Fix(input map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
I made a change to the fixer interface yesterday (merged this morning) that will cause a "doesn't satisfy the interface" conflict here but it will be easy to resolve. Just add the method:

```
func (FixerQEMUHostPort) DeprecatedOptions() []string {
	return []string{"ssh_host_port_max", "ssh_host_port_min"}
}
```

Reply:
See https://github.com/hashicorp/packer/pull/9325/files for details and other examples.